src/types/UserRef.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @see sfGuardUser::toApiReferenceArray()
```

Comment:
I don't think this comment belongs in the SDK as it refers to something in a private repository.

Reply:
Removed be1e4042d63d74a5d77e5c81b2361cd3cb652ac2